src/lib/websocket/websocket-util.ts

```diff
@@ -1,3 +1,5 @@
+import WebSocket from 'ws';
```

Comment:
```suggestion
import WebSocket from 'isomorphic-ws';
```

Without isomorphic-ws, this would cause problems in browsers